url/src/ReportRow.class.php

```diff
@@ -0,0 +1,141 @@
+<?php 
+
+class ReportRow {
+
+	public $itemID = "";
+	public $oldURL = "";
+	public $newURL = "";
+	public $transactionType = "";
+    public $resourceID = "";
+	public $currentWebAddressArray = null;
+	public $currentOnlineResource = null;
+	public $newWebAddressArray = null;
+	public $actionMessage = "";
+	public $failure = false;
+	public $updated = false;
+	private $resourceReports = [];
+
+
+	private function getFields()
+	{
+		return [
+			"Item ID" => $this->itemID,
+			"Resource ID" => $this->resourceID,
+			"Old URL" => $this->oldURL, 
+			"New URL" => $this->newURL,
+			"Transaction Type" => $this->transactionType,
+			"Current Web Address Array" => $this->getArrayValueAsString($this->currentWebAddressArray, "", "Web address is empty"),
```

Comment:
Moved the report display logic for the current and new web address array into the report class